package.json

```diff
@@ -1,19 +1,24 @@
 {
   "name": "http-traceroute",
   "version": "1.6.1",
+  "main": "lib/trace.js",
   "description": "A command line tool for following and showing HTTP redirects for a given URL",
-  "bin": "index.js",
+  "bin": {
```

Comment:
Not important, but just a tip for the future: If you want to expose a CLI with the same name as the module, you don't need to give the bin a name. You can simply write `"bin": "bin/cli.js"` - it will then default to use the name of the module